Command/RunCommand.php

```diff
@@ -140,13 +140,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         while (time() - $startTime < $maxRuntime) {
             $this->checkRunningJobs();
-
-            $excludedIds = array();
+            
             while (count($this->runningJobs) < $maxConcurrentJobs) {
                 try {
                     $this->preFindStartableJob();
                     $this->getRepository()->startTrasaction();
-                    if (null === $pendingJob = $this->getRepository()->findStartableJob($queueName, $excludedIds)) {
+                    if (null === $pendingJob = $this->getRepository()->findStartableJob($queueName)) {
```

Comment:
we are not using the feature of job dependency, however, it's supported originally. We can still keep the feature and it won't affect our use cases. And here the RunCommand class is already changed a lot based on our own application logic. So I removed this unused variable just in order to make the code more readable.